### PR TITLE
To do list web project with webpack and modular JavaScript(add and remove task feature)

### PR DESCRIPTION
**Pull request by juaryimami**
In this project:

-  I implemented the CRUD (create, read, update, delete) methods. All the elements of the user interface will be fully functional and my application is completed.
- Thank you for reviewing my project!!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+import {
+  dragStart, dragEnd, dragOver, dragDrop,
+} from './modules/dragDrop.js';
+import { listContainer, createItemTask } from './modules/renderTasks.js';
+import {
+  clearCompletedTasks, removeAllIcon, removeAllItems, removeCompletedItem, removeSelectedItem,
+} from './modules/remove.js';
 
-const toDoTasks = [
+// const listContainer = document.querySelector('.container-list');
```
Comment (cynthiainga) [inline src/index.js]: - [ ] Please kindly delete the commented line of code to have a clean and professional  codebase 😎 😉

File: src/modules/remove.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export const removeAllItems = (tasks, container) => {
+  container.innerHTML = '';
+  return [];
+};
+
+export const removeAllIcon = document.querySelector('.refresf-icon');
```
Comment (cynthiainga) [inline src/modules/remove.js:6]: - [ ] Please the refresh icon is for refresh the page and not for deleting all the task please kindly fit it 🙃

File: src/index.js [hunk 3]
```diff
@@ -20,33 +31,125 @@ const toDoTasks = [
...
+      dragDrop(e);
+      const items = [...listContainer.children];
+
+      toDoTasks = [];
+
+      items.forEach((item, index) => {
+        const newTask = new Task(item.children[1].textContent, item.children[0].checked, index);
+        toDoTasks.push(newTask);
+        setData();
+      });
```
Comment (cynthiainga) [inline src/index.js:74]: - [x] By default, new tasks should have the property index set to the value of the new array length (i.e. if a 4th task is added to the list, the index of that task should equal to 5 or the index of the 1rst task should equal to 1), the index start by 1,2,... and not 0,1,2,... Please kindly fix that 🙃 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @juaryimami, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 3 above)
Comment (juaryimami) [reply src/index.js:74]: Thank you, I will update soon.

**Review by omarsalem7 — APPROVED** [review]
Hi @juaryimami 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**